336 psw location options being read aloud as blank

duncansanderson opened the pull request:
The `vue3-simple-typeahead`  component has an accessibility issue where the options are read out, by a screen reader, as "Blank" when using the keyboard to navigation through the list of options. I've replaced this component with a `@headlessui/vue` component, which is accessible.

GemmaLouise commented on src/components/product-search/components/Autocomplete.vue:
```diff
@@ -112,6 +132,42 @@ onMounted(() => {
+.vs-autocomplete {
+    position: relative;
+
+    button {
```
Are we able to use classes here rather than element selectors? Also, can the old SCSS now be removed ?

GemmaLouise commented:
@duncansanderson I had a look at the new one and it seems to be like reeeally slow when deleting the text and trying to enter a new location - not sure if just me or an issue with the plugin?

duncansanderson commented:
@GemmaLouise - I've now replaced the `@headlessui/vue` component with another component called `vue-combo`blocks`, which appears to be much faster and a little easier to work with. I've also removed SCSS for the original component. Could you re-review these changes please?
